reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
+---------------
 
 For compiling under Windows, Linux or OSX, the following is required:
 
--  Python 2.7+ (3.0 is untested as of now).
--  SCons build system.
+-  Python 2.7+ [3.0 is untested as of now]
+-  pywin32 [Optional, Windows only. Applied when -j(number of cores) specified]
+-  SCons build system
 -  Android SDK version 19 [Note: Please install all Tools and Extras of sdk manager]
 -  Android build tools version 19.1
--  Android NDK
--  Gradle
--  OpenJDK 6 or later (or Oracle JDK 6 or later)
+-  Android NDK r10e or later
+-  Gradle [will be downloaded and installed when compiling for android if it doesn't exist]
+-  JDK 6+ (Oracle's releases) or OpenJDK 6+
+
+B. Setting up Build System
+--------------------------------------------------------
+
+1) Scons : please, refer to their respective docs
```

Comment:
Well my point is that referring users to the SCons documentation does not make sense here. The SCons documentation is for people who need to develop a buildsystem using SCons. I think you're right to rename the section from `Setting up SCons` to `Setting up the buildsystem`, but then there's no need to mention SCons at all. What the docs should mention is what the users need to do.

> NDK r10e or later? Afaik, NDK older than r10e doesn't have arm-linux-androideabi-4.9 (while detect.py is now at 4.9), instead they have arm-linux-androideabi-4.8

No problem with that :)